tests/data/output/schema/formula/input-schema.yml

```diff
@@ -10,9 +10,6 @@ properties:
 - fqdn:
     type: hostname
     formula: host ~ `.` ~ domain
-
-# this is effectively a const but we only get to know that at runtime,
-# this tests serde edge case where the yaml Value is not a serde_yaml::Value::String
 - priority:
     type: integer
-    formula: -7
+    formula: "-7"
```

Comment:
this may be worth running through Lucian and others to see if this is expected, or would it be surprising

Reply:
Fixed by the latests commit in this branch.